src/libs/geometry/src/geom_static.cpp

```diff
@@ -9,6 +9,8 @@ Import library main file
 ******************************************************************************/
 #include "geom.h"
 
+#include <fmt/format.h>
```

Comment:
I think we have `<format>` available unless you want some features from the latest fmt version...

Reply:
Is it finally here? 🥳

Reply:
Looks like we can't quite use `std::format` for our clang builds?

Reply:
Oh..may be
We should definitely update the pipeline

Reply:
I reverted back to using `fmt::format` for now. I suggest updating the pipeline and replace all usages of `fmt::format` later at the same time.